README.md

```diff
@@ -249,7 +249,62 @@ module.exports = {
     let checker = VersionChecker.forProject(this.project);
 
     for (let { name, root } = checker.allAddons()) {
-      // access to the add-on, in this case root + name
+      // access to the addon, in this case root + name
```

Comment:
Overly pedantic, I know, but just keeping the order consistent with the destructuring.

```suggestion
      // access to the addon, in this case name and root
```